.github/workflows/application.yaml

```diff
@@ -24,22 +23,23 @@ permissions:
   pull-requests: write
 
 jobs:
-  build:
+  docker_build_push_ecr:
     strategy:
-      fail-fast: false
+      fail-fast: true
       matrix:
         {% for env in matrix_envs -%}
         - environment: @{{ env.name }}
           stack: app
         {% endfor %}
 
-    uses: tx-pts-dai/github-workflows/.github/workflows/docker-build-push-ecr.yaml@v0.9.0
+    uses: tx-pts-dai/github-workflows/.github/workflows/docker-build-push-ecr.yaml@v0.23.0
     with:
       environment: ${{ matrix.stack }}-${{ matrix.environment }}
+      docker_push: ${{ github.ref_name == github.event.repository.default_branch || github.event_name == 'pull_request' }}
       docker_context: application
+      image_tag: ${{ github.sha }}
```

Comment:
it can be a default in the shared workflow

Reply:
github.sha is automatically tagged in the tx-pts-dai/github-workflows/.github/workflows/docker-build-push-ecr.yaml@v0.23.0 therefore it can be removed here

Reply:
Todo: remove it